pkg/controllers/reconciler_test.go

```diff
@@ -61,18 +67,35 @@ func TestUpgrade(test *testing.T) {
 			expectedInitimage: initimage,
 			upgrade:           false,
 		},
+		{
+			envVars: map[string]string{strings.ReplaceAll(strings.ToUpper(filepath.Base(image)), "-", "_") + "_SHA": image + "@sha256:" + sha256,
+				strings.ReplaceAll(strings.ToUpper(filepath.Base(initimage)), "-", "_") + "_SHA": initimage + "@sha256:" + sha256},
```

Comment:
This is rather complex for the reader to understand what this test is all about. I'd suggest to spell out the strings for readability, maybe with a different image name:
```suggestion
			envVars: map[string]string{
			        INTEL_QAT_PLUGIN_SHA: "intel/intel-qat-plugin@sha256:000000000000000000000000000000000000000000000000000000000000000a",
			        INTEL_QAT_INITIMAGE_SHA: "intel/intel-qat-initimage@sha256:000000000000000000000000000000000000000000000000000000000000000a",
				},
```

Reply:
If you see the image name, it is fixed as dsa plugin's image name. Is the qat example just an example?